DevEdu.API/DevEdu.DAL/Repositories/CourseRepository.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Dapper;
+
+namespace DevEdu.DAL.Repositories
+{
+    public class CourseRepository : BaseRepository, ICourseRepository
+    {
+        public CourseRepository()
+        {
+                
+        }
+
+        public void AddTagToTopic(int topicId, int tagId)
+        {
+            _connection.Query(
+                "dbo.Tag_Topic_Insert",

```

Comment:
each name of stored procedures must be placed somewhere as a constant
otherwise, it's an example of the "magic numbers" antipattern